### PR TITLE
Add up-to-date Python template

### PR DESCRIPTION
**Pull request by mkenigs**
This works with no modifications in flox-examples/hello-python. I don't think we've resolved where `default.nix` should go, but hey this works

Related: https://github.com/flox/product/issues/14

### CODE REVIEW

File: templates/python/flake.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    capacitor args (
+      {
+        inputs,
+        self,
+        ...
+      }: let
+        outputsWithoutFloxEnv = {
+          description = "Python template";
+          config = {
+            stabilities = {
```
Comment (tomberek) [inline templates/python/flake.nix:25]: @ysndr is this equivalent to the nixpkgs plugin?

Comment (ysndr) [reply templates/python/flake.nix:25]: If youre speaking of the nixpkgs plugin we have on the `plugin` branch of floxpkgs [1], then yes

[1]: https://github.com/flox/floxpkgs/blob/plugins/plugins/stabilities.nix

**Comment by tomberek** [comment]
https://github.com/flox/product/issues/14

File: templates/python/flake.nix (hunk 1 above)
Comment (mkenigs) [reply templates/python/flake.nix]: Added that plugin instead, look good now?

**Pull request merged**